qml/pages/FirstPage.qml

```diff
@@ -23,6 +24,46 @@ Page {
                            { gpsDataSource: page.gpsDataSource, compass: page.compass})
         }
     }
+    Item {
+            id: gpsTimes
+            property date activateGPSTimestamp: new Date()
+            property date firstFixTimestamp: new Date()
+            property bool pendingFix : true
+            property int secsSincePosition : Math.round((new Date() - positionSource.position.timestamp)/1000)
+
+            property int secs2FF : 0
+            property string secs2FFStr: roundElapsedTime(secs2FF)
+            property string tTFF: {
+                if ( pendingFix) {
+                    if (positionSource.position.coordinate.isValid && (positionSource.position.timestamp > activateGPSTimestamp)) {
+                        pendingFix=false
+                        t2secs2FF(positionSource.position.timestamp)
+                        //firstFixTimestamp=positionSource.position.timestamp
+                        //Math.round((firstFixTimestamp - activateGPSTimestamp)/1000)
+                        Notices.show("GPS Time to First Fix "+secs2FFStr, Notice.Long)
+                    } else {
+//                        t2secs2FF(new Date())
+                    }
+                }
+                return secs2FFStr
+            }
+            function t2secs2FF(T){
+                secs2FF = Math.round((T - activateGPSTimestamp)/1000)
+            }
```

Comment:
One line functions are usually not worth it. This is used only once (twice) so I think it can be inlined.